### PR TITLE
feat: Add paragraphs query options

### PR DESCRIPTION
**Pull request by BadMask121**
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update or tests (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) doc
- [ ] I have added tests and/or examples that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by jcdyer** [comment]
Hi @BadMask121!  Thanks for your contribution.  I'll add some docs to these methods, and get them out the door in our 0.5 release.

File: src/transcription/prerecorded/options.rs [hunk 1]
```diff
@@ -24,6 +24,9 @@ pub struct Options {
     search: Vec<String>,
     replace: Vec<Replace>,
     keywords: Vec<Keyword>,
+    filler_words: bool,
+    smart_format: bool,
+    paragraphs: Option<bool>,
```
Comment (jcdyer) [inline src/transcription/prerecorded/options.rs:29]: Can you change `filler_words` and `smart_format` to `Option<bool>`, so they don't get serialized unless the user requests them?

**Review by jcdyer — CHANGES_REQUESTED** [review]
Oops. Spoke too soon. Some tests are failing. See below.

**Comment by DamienDeepgram** [comment]
This has been addressed in this PR: https://github.com/deepgram-devs/deepgram-rust-sdk/pull/76

**Pull request closed without merging**